install/0000_80_machine-config-operator_00_clusterreader_clusterrole.yaml

```diff
@@ -33,3 +33,15 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - config.openshift.io
+    resources: 
+      - images 
```

Comment:
Other than featuregates does the other items need to be explicit? How were they working before?

Reply:
uhhhh, I don't think they were working before. FG seems to required images, clusterversions, featuregates. So at least those here. It seems the actual FgAccessor was only used in the controller before. Especially because the configInformer was not used in the operator